src/RealFourierArray.jl

```diff
@@ -27,7 +27,9 @@ mutable struct RealFourierArray{Treal <: Real, T <: Union{Treal, Complex{Treal}}
     _state::RFA_state
 end
 # Type of the real part
-Base.eltype(A::RealFourierArray{Treal, T}) where {Treal, T} = T
+Base.eltype(::RealFourierArray{Treal, T}) where {Treal, T} = T
+Base.size(array::RealFourierArray) = size(array._real)
+Base.size(array::RealFourierArray, i) = size(array._real, i)
```

Comment:
I'm not sure we should do this (the size), since it's ambiguous whether that refers to the real or fourier part

Reply:
Doesn't matter since it's the same.

Reply:
Ah right.